ci.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -ex
+cd "$(dirname "$0")"
+
+repo_root=$(readlink -f $(dirname "${BASH_SOURCE[0]}"))
+
+if [ -z $TRAVIS_RUST_VERSION ]; then
+    echo "Expected TRAVIS_RUST_VERSION to be set in env"
+    exit 1
+fi
+
+python3 -m pip install -r ./mbedtls-sys/vendor/scripts/basic.requirements.txt
```

Comment:
Can this use a python virtualenv instead of installing directly on the system?

Reply:
updated